test/TesterInternal/General/LoggerTest.cs

```diff
@@ -1,478 +0,0 @@
-        }
-
-        [Fact, TestCategory("Functional"), TestCategory("Logger")]
-        public void Logger_SeverityLevelTest()
-        {
-            LoggerImpl target = LogManager.GetLogger("Test");
-            Assert.Equal(Severity.Verbose,  target.SeverityLevel);  //  "Default severity level not calculated correctly for runtime logger with no overrides"
-            target.SetSeverityLevel(Severity.Verbose2);
-            Assert.Equal(Severity.Verbose2,  target.SeverityLevel);  //  "Custom severity level was not set properly"
-            LogManager.SetRuntimeLogLevel(Severity.Warning);
-            Assert.Equal(Severity.Verbose2,  target.SeverityLevel);  //  "Severity level was re-calculated even though a custom value had been set"
-        }
-
-
-        [Fact, TestCategory("Functional"), TestCategory("Logger")]
-        public void Logger_GetLoggerTest()
-        {
-            Logger logger = LogManager.GetLogger("GetLoggerTest", LoggerType.Runtime);
-            Assert.NotNull(logger);
-
-            Logger logger2 = LogManager.GetLogger("GetLoggerTest", LoggerType.Runtime);
-            Assert.NotNull(logger2);
-            Assert.True(ReferenceEquals(logger, logger2));
-
-            Logger logger3 = LogManager.GetLogger("GetLoggerTest", LoggerType.Application);
-            Assert.NotNull(logger3);
-            Assert.True(ReferenceEquals(logger, logger3));
-        }
-
-        [Fact, TestCategory("Functional"), TestCategory("Logger")]
-        public void Logger_CreateMiniDump()
-        {
-            var dumpFile = LogManager.CreateMiniDump();
-            output.WriteLine("Logger.CreateMiniDump dump file = " + dumpFile);
-            Assert.NotNull(dumpFile);
-            Assert.True(dumpFile.Exists, "Mini-dump file exists");
-            Assert.True(dumpFile.Length > 0, "Mini-dump file has content");
-            output.WriteLine("Logger.CreateMiniDump dump file location = " + dumpFile.FullName);
-        }
-
-        [Fact, TestCategory("Functional"), TestCategory("Logger")]
-        public void Logger_AddRemoveOverride()
-        {
-            const string name = "LoggerOverrideTest";
-            const string fullName = "Runtime." + name;
-            var logger = LogManager.GetLogger(name, LoggerType.Runtime);
-            var initialLevel = logger.SeverityLevel;
-
-            LogManager.AddTraceLevelOverride(fullName, Severity.Warning);
-            Assert.Equal(Severity.Warning,  logger.SeverityLevel);  //  "Logger level not reset after override added"
-
-            LogManager.RemoveTraceLevelOverride(fullName);
-            Assert.Equal(initialLevel,  logger.SeverityLevel);  //  "Logger level not reset after override removed"
-        }
-
-        [Fact, TestCategory("Functional"), TestCategory("Logger")]
-        public void Logger_BulkMessageLimit_DifferentLoggers()
```

Comment:
none of these tests are useful for the legacy logging package?

Reply:
legacy logging has its own tests. Maybe we can add some message bulking tests from this file to that. But most of these is not relevant I think